src/dispatch/static/dispatch/src/router/config.js

```diff
@@ -298,9 +298,15 @@ export const protectedRoute = [
         {
           path: "/:organization/feedback",
           name: "FeedbackTable",
-          meta: { title: "Feedback" },
+          meta: { title: "Incident feedback", group: "feedback" },
           component: () => import("@/feedback/Table.vue"),
         },
+        {
+          path: "/:organization/service-feedback",
+          name: "ServiceFeedbackTable",
+          meta: { title: "Service feedback", group: "feedback" },
+          component: () => import("@/service_feedback/Table.vue"),
```

Comment:
I'd organize the Vue.js code the same we organize the backend code. One directory called `feeback`, which already exists, with two subdirectories called `incident` and `service`. Then here you'd do:

```suggestion
          component: () => import("@/feedback/service/Table.vue"),
```